taqueria-vscode-extension/src/lib/helpers.ts

```diff
@@ -1105,6 +1105,15 @@ export class VsCodeHelper {
 			}
 		}
 		if (fileSelectionBehavior === 'currentFile') {
+			let viewColumn = this.vscode.window.activeTextEditor?.viewColumn;
```

Comment:
Sidebars and panels do not have multi-column layout, so that `viewColumn` is undefined for that.
I've used that to detect if the `activeTextEditor` is a proper text editor window.

Reply:
This is weird that VsCode lacks a canonical way of doing this.